edvart/report_sections/multivariate_analysis.py

```diff
@@ -628,15 +639,15 @@ def parallel_coordinates(
 
         if is_categorical_color:
             categories = df[color_col].unique()
-            colorscale = list(discrete_colorscale(len(categories)))
+            colorscale = get_default_discrete_colorscale(n_colors=len(categories))
             # encode categories into numbers
             color_series = pd.Series(pd.Categorical(df[color_col]).codes)
         else:
             color_series = df[color_col]
-            colorscale = "Bluered_r"
+            colorscale = "Bluered"
```

Comment:
Why reversing?

Reply:
I was just playing with and forgot the change there. tbh, I'm not sure, why the colors are reversed in the first place, but it doesn't really matter.